llama-index-integrations/llms/llama-index-llms-bedrock-converse/llama_index/llms/bedrock_converse/utils.py

```diff
@@ -60,21 +66,61 @@
     "cohere.command-r-plus-v1:0",
     "mistral.mistral-large-2402-v1:0",
     "mistral.mistral-large-2407-v1:0",
+    "meta.llama3-1-8b-instruct-v1:0",
+    "meta.llama3-1-70b-instruct-v1:0",
+    "meta.llama3-2-1b-instruct-v1:0",
+    "meta.llama3-2-3b-instruct-v1:0",
+    "meta.llama3-2-11b-instruct-v1:0",
+    "meta.llama3-2-90b-instruct-v1:0",
```

Comment:
Thought: I wonder if there's some way to ping an API and auto-populate these lists..

Reply:
@logan-markewich Probably not without using the AWS API, which would require active/valid IAM credentials: https://docs.aws.amazon.com/bedrock/latest/userguide/models-get-info.html#:~:text=Use%20the%20API%20to%20retrieve%20information%20about%20Amazon,that%20Amazon%20Bedrock%20provides%2C%20send%20a%20ListFoundationModels%20request.